test/multiverse/suites/unicorn/unicorn_test.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'unicorn'
+require 'newrelic_rpm'
+
+class UnicornTest < Minitest::Test
+  include MultiverseHelpers
+  attr_accessor :server
+
+  def setup
+    File.truncate('log/newrelic_agent.log', 0)
```

Comment:
My hunch is that this operation is more costly than `File.unlink`. Is there a risk of losing the write access required to replace the deleted file that has led us to truncate instead of deleting?

Reply:
I didn't know of `File.unlink`'s existence! I'll give that a try.